pkg/kvstore/etcd.go

```diff
@@ -1138,9 +1185,15 @@ func (e *etcdClient) Status() *models.Status {
 
 // GetIfLocked returns value of key if the client is still holding the given lock.
 func (e *etcdClient) GetIfLocked(ctx context.Context, key string, lock KVLocker) (bv []byte, err error) {
-	defer func() {
-		Trace("GetIfLocked", err, logrus.Fields{fieldKey: key, fieldValue: string(bv)})
-	}()
+	if traceEnabled {
```

Comment:
Any reason for adding the `traceEnabled` check? The performance gain should be mostly negligible, at the cost of increased complexity. Same for all the other occurrences.